FIAT/expansions.py

```diff
@@ -24,11 +23,22 @@ def morton_index3(p, q=0, r=0):
 def jrc(a, b, n):
     """Jacobi recurrence coefficients"""
     an = (2*n+1+a+b)*(2*n+2+a+b) / (2*(n+1)*(n+1+a+b))
-    bn = (a*a-b*b) * (2*n+1+a+b) / (2*(n+1)*(2*n+a+b)*(n+1+a+b))
+    bn = (a+b)*(a-b)*(2*n+1+a+b) / (2*(n+1)*(n+1+a+b)*(2*n+a+b))
```

Comment:
Is this for numerical accuracy?

Reply:
no, mainly aesthetics and saves one multiplication

Reply:
When a and b are close together, then a^2-b^2 is probably better than (a+b)(a-b) if a, b are large but worse if they are small?  At any rate, a = b + \epsilon is not a use case as far as I'm aware?

Reply:
They're both integers, moreover b \in {0, 1}